### ISSUE TITLE
Autocompletion not working with mf_insert_event()

### ISSUE THREAD

**Issue by magneto538** [nojanath/SublimeKSP#59]

**Comment by mkruselj** [comment]
That's because that function is defined as requiring a return value in ksp_builtins_data.py. Same for mf_insert_file(). The way to solve this is opening ksp_plugin.py, then around line 220 there's this:

```
# for a certain set of functions with return values one often discards the return value
# add them to the second set too
```

Just add functions that require a return value, but you want to autocomplete even without one (meaning - they still work even though assigning them to a variable is not necessary), here. Make sure you don't use () in the names of functions. Boom, fixed!

Here's how it looks over here:

`for func in ['_get_engine_par','_get_engine_par_disp','_get_folder','_num_slices','_pgs_get_key_val', '_set_engine_par', '_slice_idx_loop_end','_slice_idx_loop_start','_slice_length','_slice_loop_count','_slice_start','abs','by_marks','find_group','find_mod','find_target','get_control_par','get_control_par_arr','get_control_par_str','get_engine_par','get_engine_par_disp','get_engine_par_disp_m','get_event_par','get_event_par_arr','get_folder','get_sample_length','group_name','load_ir_sample','mf_get_buffer_size','mf_get_byte_one','mf_get_byte_two','mf_get_channel','mf_get_command','mf_get_length','mf_get_note_length','mf_get_num_tracks','mf_get_pos','mf_get_track_idx', 'mf_insert_event','mf_insert_file', 'ms_to_ticks','num_slices','num_slices_zone','output_channel_name','pgs_get_key_val','pgs_get_str_key_val','play_note','random','search', 'set_engine_par', 'sh_left','sh_right','slice_idx_loop_end','slice_idx_loop_start','slice_length','slice_loop_count','slice_start','ticks_to_ms','zone_slice_idx_loop_end','zone_slice_idx_loop_start','zone_slice_length','zone_slice_loop_count','zone_slice_start','array_equal']`

**Comment by neblix** [comment]
Fixed in 946bbce
